Yoast/wordpress-seo
# JSON-LD breadcrumbs

## Summary

This PR can be summarized in the following changelog entry:

* Adds support for JSON-LD breadcrumbs. Props [teolaz](https://github.com/teolaz)

## Test instructions

This PR can be tested by following these steps:

* Enable breadcrumbs
* Open a post or something else what isn't the frontpage
* View the page source and see JSON-LD output in de `<head>` 
* In the HTML-breadcrumbs the RFDa breadcrumb structure should be removed.

## Quality assurance


Replaces #9705
